tests/test_models.py

```diff
@@ -0,0 +1,43 @@
+from django.test import TestCase
+
+import gdpr_assist
+from .gdpr_assist_tests_app.models import (
+    ForeignKeyFieldModel,
+    ModelWithPrivacyMeta,
+)
+
+
+class AnonimisationTreeTestCase(TestCase):
+    def test_valid_fk_Field(self):
+        tree = ForeignKeyFieldModel.get_anonymisation_tree()
```

Comment:
how does this work?

Reply:
What part do you need explanation on? Happy to do a walkthrough tomorrow if we can find time to sync up.